Folder-Sync.md

```diff
@@ -4,34 +4,45 @@ title: Folder Sync
 legacyUrl: /support/solutions/articles/44001162190-folder-sync
 ---
 
-Folder sync allows you to copy content to folders and external hard drives, for backup as well as archiving in multiple resolutions.
+Folder sync allows you to copy content to folders and external hard drives, for backup as well as archiving in multiple resolutions. This is an [Emby Premiere](Emby-Premiere.md) feature.
 
-When content is archived in multiple resolutions, Emby apps will automatically choose the version that is most efficient for them. This will help relieve stress on your server's CPU.
+When content is archived in multiple resolutions, Emby apps will automatically choose the version that is most efficient for them. This will help relieve stress on your server's CPU. The availability of these versions would depend on the permissions granted when setting up Folder Sync.
 
 ## Installation
 
-To install folder sync, open the **Emby Server Dashboard** and navigate to **Sync** -> **Services**. 
-
-Click on Folder Sync underneath **Available Services**.
+To install Folder Sync, click the **Plugins** menu option on the left from your web Dashboard. Then click the **Catalog** option up top on the right side. Scroll down until you find the **Sync** section. Click on the **Folder Sync** plugin and select **Install**.
```

Comment:
I will change it to 
`To install Folder Sync, open the **Emby Server Dashboard** and navigate to **Advanced** and click on **Plugins**. Open the **Catalog** and scroll down until you find the **Sync** section. Click on the **Folder Sync** plugin and select **Install**. See [Plugins Overview](Plugins.md).`

Initially I avoided mentioning Dashboard because we have a **Dashboard** button on what you refer to as the **Emby Server Dashboard** and clicking the Dashboard button gives you a view of the dashboard which is not what you are referring as the dashboard. 

Anyway if the standard is to refer to the  view that you get after clicking the server cogwheel button in top right corner as the Emby Server Dashboard then I will be in line.

Reply:
The "Server Dashboard" is the "entry point" to server settings. Once you are there, you will see all the menus, so it's still a good orientation point, even though it's also a specific view.